### ISSUE TITLE
the implementation of the yolov3

### ISSUE THREAD

**Issue by 1453042287** [ShuangXieIrene/ssds.pytorch#44]
i read the code, but i found that the loss function is ssd's? the multibox loss function is't the function in yolo's paper, right?

**Comment by 1453042287** [comment]
@seongkyun thanks a lot but i wonder if the performance is as good as the 'real' yolov3's

**Comment by seongkyun** [comment]
Exactly yes. You know that there is any code that can train Yolo V3 with any other deep learning toolkit except Darknet. Every other codes are just reads pre-trained weight file and does forward pass.
